client/src/components/Indicators.js

```diff
@@ -406,6 +409,10 @@ class IndicatorsWithoutI18n extends Component {
                   </button>
                   <IndicatorsViz {...this.state} />
                   <button
+                    aria-hidden={
+                      this.state.xAxisStart + this.state.xAxisViewableColumns >= xAxisLength ||
+                      this.state.activeTimeSpan
+                    }
```

Comment:
Errr wait, one thing I'm realizing here though is that ... uh ... you're not in TypeScript.  The TypeScript definitions for `aria-hidden` are that it be either a boolean, undefined, `"true"` or `"false"` but it seems like `this.state.activeTimeSpan` can be a string like `"year"`, in which case you might inadvertently be setting the actual value of the `aria-hidden` attribute to `"year"`, which would make no sense to browsers.  You might want to check to see if React is converting the string value to `"true"` under-the-hood, in which case you don't need to worry, or if it's just passing it through to the underlying DOM value, in which case it will confuse screen readers.

Reply:
Er the alternative of course is just to change `this.state.activeTimeSpan` to `!!this.state.activeTimeSpan` so it's coerced to a boolean.

Ah this makes me miss TypeScript!!

Reply:
ooooh so this is actually a mistake! We're actually checking to see if `this.state.activeTimeSpan` is equal to "year", not checking its existence.